configuration/cdap-site.xml

```diff
@@ -893,19 +1096,27 @@
   <property>
     <name>explore.active.operation.timeout.secs</name>
     <value>86400</value>
-    <description>Timeout value in seconds for a SQL operation whose result is not fetched completely</description>
+    <description>
+      Timeout value in seconds for an SQL operation whose result was not
```

Comment:
Actually, as "SQL" is (by ANSI standard) pronounced spelt out as "ess-que-ell", it should be "an".